networking/networking-operators-overview.adoc

```diff
@@ -23,3 +23,7 @@ When you create your {product-title} cluster, pods and services running on the c
 [id="networking-operators-overview-external-dns-operator"]
 == External DNS Operator
 The External DNS Operator deploys and manages ExternalDNS to provide the name resolution for services and routes from the external DNS provider to {product-title}. For more information, see xref:../networking/external_dns_operator/understanding-external-dns-operator.adoc#external-dns-operator[Understanding the External DNS Operator].
+
+[id="ingress-node-firewall-operator-1"]
+== Ingress Node Firewall Operator
+The Ingress Node Firewall Operator uses an extended Berkley Packet Filter (eBPF) and eXpress Data Path (XDP) plugin to process node firewall rules, update statistics and generate events for dropped traffic. The operator manages ingress node firewall resources, verifies configuration,avoids miss configured rules that can lock up cluster access, and load ingress node firewall XDP program to the selected interfaces in the rule's object(s). For more information, see xref:../networking/ingress-node-firewall-operator.adoc#ingress-node-firewall-operator[Understanding the Ingress Node Firewall Operator]
```

Comment:
```suggestion
The Ingress Node Firewall Operator uses an extended Berkley Packet Filter (eBPF) and eXpress Data Path (XDP) plugin to process node firewall rules, update statistics and generate events for dropped traffic. The operator manages ingress node firewall resources, verifies firewall configuration, avoids incorrectly configured rules that can prevent cluster access, and loads ingress node firewall XDP programs to the selected interfaces in the rule's object(s). For more information, see xref:../networking/ingress-node-firewall-operator.adoc#ingress-node-firewall-operator[Understanding the Ingress Node Firewall Operator]
```
A few corrections here. One question - is there a better word to use than "avoids"? Does the operator fix incorrectly configured rules automatically, or alert the customer to them, or disable incorrect rules when it finds them? I'm not sure what "avoid" means in this context.

Reply:
What about `prevents`? s/avoids/prevents

Reply:
As we have error messages if admin tries to create rules that block cluster access, it can be stated - **does not allow incorrectly configured rules**. If there is an validation error with rule creation, rule is not created at all.